.pre-commit-config.yaml

```diff
@@ -20,16 +21,17 @@ repos:
       - id: check-merge-conflict
       - id: check-toml
       - id: debug-statements
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v4.0.0-alpha.8
+  - repo: local
     hooks:
       - id: prettier
+        name: prettier
+        entry: npx prettier --write
+        language: node
         types_or:
           [javascript, jsx, ts, tsx, json, scss, sass, css, yaml, markdown]
-        args:
-          - --no-config
-          - --no-semi
       - id: prettier
+        entry: npx prettier --write --plugin=prettier-plugin-django-alpine "**/*.html"
+        language: node
```

Comment:
With moving it to the local hook you don't have to have this as a separate step anymore, you can just rely on the prettier config for loading the plugin and addressing all of the file types.

Reply:
You're correct about this, although I did have to add `html` to the `types_or` argument for it to actually check django templates